src/admin-views/aggregator/settings.php

```diff
@@ -489,10 +489,20 @@
 		),
 	);
 
+	// Ensure that "(do not override)" is set up for Eventbrite import statuses, and "Published" is not.
+	$eventbrite_origin_post_statuses = array( 'do_not_override' => esc_html__( '(do not override)', 'the-events-calendar' ) ) + $origin_post_statuses;
```

Comment:
I've fixed the one-liner, but I'm using `+` here because we want this status to be the first item in the array. Neither `array_merge` nor `array_unshift` quite fit what we want, so we use the `+`.

It is used in some other locations in the EA code as well for the same purpose.